api/users.go

```diff
@@ -246,5 +255,13 @@ func (self *ApiServer) SetUserRoles(
 	// users.AddUserToOrg
 	err = users.AddUserToOrg(ctx, users.UseExistingUser,
 		principal, in.Name, []string{in.Org}, acl)
+
+	services.LogAudit(ctx,
+		org_config_obj, principal, "user_grant",
+		ordereddict.NewDict().
+			Set("Username", in.Name).
+			Set("Roles", in.Roles).
```

Comment:
we probably should log all the permissions as well because the call could include additional permissions which are not roles.